src/collection-facets/facets-template.ts

```diff
@@ -85,6 +92,7 @@ export class FacetsTemplate extends LitElement {
   private getFacetsTemplate(facetGroup: FacetGroup): TemplateResult {
     let facetsBucket = facetGroup?.buckets as FacetBucket[];
 
+    if (!facetsBucket) return html`${nothing}`;
```

Comment:
Q: why is this file also committed?

Reply:
there's a failing test related to it, i added this part as fix

Reply:
will fix the failing test on a different PR